ostelco-ios-client/ModelControllers/ESimManager.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  ESimManager.swift
+//  ostelco-ios-client
+//
+//  Created by mac on 9/27/19.
+//  Copyright © 2019 mac. All rights reserved.
+//
+
+import CoreTelephony
+import PromiseKit
+import ostelco_core
+
+class ESimManager {
+    static let shared = ESimManager()
```

Comment:
Yes, like this! Why is it a singleton though?

Reply:
That's they only pattern I know 😎

Reply:
Hmm, maybe we can leave it for now and I'll change it with the stage decider split.